docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// the alignment of the usings with the methods is intentional so they can display on the same level in the docs site. 
+        using Microsoft.ML.Runtime.Api;
+        using Microsoft.ML.Runtime.Data;
+        using Microsoft.ML.Trainers;
+        using System.Collections.Generic;
```

Comment:
you can align them correctly, since you are taking all the file.

Reply:
Ok. The prepended space will be removed.